<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat_button</class>
 <widget class="QWidget" name="chat_button">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>60</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>60</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>60</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="button_username">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>2</y>
     <width>320</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QLabel" name="button_Icon">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>5</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="res.qrc">:/res/head.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="button_chat">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>25</y>
     <width>320</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>button</zorder>
  <zorder>button_username</zorder>
  <zorder>button_Icon</zorder>
  <zorder>button_chat</zorder>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
